src/Tribe/Views/V2/View.php

```diff
@@ -2225,11 +2256,8 @@ protected function setup_additional_views( array $events = [], array $template_v
 		 */
 		$recent_past_threshold = apply_filters( "tribe_events_views_v2_threshold_to_show_recent_past_events", 0, $events, $template_vars, $this );
```

Comment:
Move the `absint` use here, on the filter return value, to make sure any operation done on the `$recent_past_threshold` variable will use a positive integer.